engine/api/latest/index.html

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This file handles the redirect from `/engine/api/latest/` to the real latest version, so that the user always lands on a versioned page.

Reply:
I was thinking if you'd rather not use a redirect you could potentially have this file do:

`{% include_relative {{ latest_stable_docker_engine_api_version }}/index.md %}`

Otherwise this approach LGTM!

Reply:
The redirect works, but the API doc itself is a 404 (may be expected; https://deploy-preview-4892--docker-docs.netlify.com/engine/api/v1.32/)

Should we use `latest` or `current` ?

Reply:
Yes, that's expected. I think `latest` is clearer, but I'll wait for what y'all think.

I really don't mind using a redirect. I'd rather the user end up on a versioned page rather than have them bookmark `latest` and then wonder why it changed underneath them.

Reply:
I'm +1 for using a redirect as well; it also helps, as people automatically get a "permalink" if they bookmark, or share.